randum.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>;
+#include <stdlib.h>;
+#include <time.h>;
```

Comment:
Please see us about your code - you didn't need to use a switch for this assignment.  The random number generated also isn't between 1 and 10.